src/Umbraco.Infrastructure/Services/Notifications/TemplateSavedNotification.cs

```diff
@@ -17,12 +17,24 @@ public TemplateSavedNotification(IEnumerable<ITemplate> target, EventMessages me
         {
         }
 
-        public TemplateSavedNotification(ITemplate target, EventMessages messages,
-            Dictionary<string, object> additionalData) : base(target, messages) => AdditionalData = additionalData;
-
-        public TemplateSavedNotification(IEnumerable<ITemplate> target, EventMessages messages,
-            Dictionary<string, object> additionalData) : base(target, messages) => AdditionalData = additionalData;
+        public bool? CreateTemplateForContentType
```

Comment:
I guess this should just be bool, and default to false

Reply:
Ah true, I change it so it just defaults to false